CHANGELOG.md

```diff
@@ -3,7 +3,12 @@ CHANGELOG
 
 master
```

Comment:
```suggestion
master
------
* todo...

v2.0.0
------
* Drop support for PHP ^7.3 || ^8.0, support now only ^8.1
* Force usage of native sprintf function
* Added rule to ban shell execution via backticks
* Added rule to ban print statements
* Allow Composer plugin ergebnis/composer-normalize
* Add Composer keyword for asking user to add this package to require-dev instead of require
* Normalize leading backslashes in banned function names
```